kotlinpoet/build.gradle.kts

```diff
@@ -23,11 +24,12 @@ spotless {
   kotlin {
     targetExclude(
       // Non-Square licensed files
-      "src/commonMain/kotlin/com/squareup/kotlinpoet/ClassName.kt",
-      "src/commonTest/kotlin/com/squareup/kotlinpoet/AbstractTypesTest.kt",
-      "src/commonTest/kotlin/com/squareup/kotlinpoet/ClassNameTest.kt",
-      "src/commonTest/kotlin/com/squareup/kotlinpoet/TypesEclipseTest.kt",
-      "src/commonTest/kotlin/com/squareup/kotlinpoet/TypesTest.kt",
+      "src/*Main/kotlin/com/squareup/kotlinpoet/ClassName.kt",
+      "src/*Main/kotlin/com/squareup/kotlinpoet/ClassName.*.kt",
+      "src/*Test/kotlin/com/squareup/kotlinpoet/AbstractTypesTest.kt",
+      "src/*Test/kotlin/com/squareup/kotlinpoet/ClassNameTest.kt",
+      "src/*Test/kotlin/com/squareup/kotlinpoet/TypesEclipseTest.kt",
+      "src/*Test/kotlin/com/squareup/kotlinpoet/TypesTest.kt",
```

Comment:
There seems to be a fair amount of testing that is JVM-only, and some that are incompatible with each other—such as the part about numbers.
I think maybe it could be kept in the JVM for now and then migrated gradually?

Reply:
Given that the overall strategy is to move everything to JVM first, I think it does makes sense for the tests to follow suit. There are indeed tests that are JVM-only and will stay in the JVM source set even after we've extracted common code back into `common`.

Reply:
Understood, so it's: not `*` but `jvmTest`? I'll change it and push it out later~

Reply:
Okay. Got it~